src/Germany/Mews.Fiscalizations.Germany/FiskalyClient.cs

```diff
@@ -56,21 +61,21 @@ public Task<ResponseResult<Tss>> GetTssAsync(AccessToken token, Guid tssId)
             );
         }
 
-        public Task<ResponseResult<Tss>> CreateTssAsync(AccessToken token, TssState state, string description = null)
+        public Task<ResponseResult<CreateTss>> CreateTssAsync(AccessToken token)
         {
-            return Client.ProcessRequestAsync<Dto.CreateTssRequest, Dto.TssResponse, Tss>(
+            return Client.ProcessRequestAsync<object, Dto.CreateTssResponse, CreateTss>(
                 method: HttpMethod.Put,
                 endpoint: $"tss/{Guid.NewGuid()}",
```

Comment:
Didn't we agree that we will try to create TSS with the same ID as the original ones? Which means we need the ability to provide tssID, rather than getting a random GUID. cc @marektresnak as you're the one who requested him to remove it 😄

Reply:
Yea that's what we said, but after looking at the usages in Mews, i saw no benefit from doing that.

Having the same TSS ID in both API versions doesn't bring any benifit.

My idea is to create a new TSS whenever there is an update on an enabled integration. (which makes sense, as you can update an integration with new API/Secret keys).

Reply:
Is there a benefit in creating a new TSS every time? It will make the dashboard look horrible.
There is a hidden benefit in having the TSS with the same ID. The simplicity. You have 1 Tss. And it's one ID. forever has been.

Reply:
I think it does make sense to create a new TSS each time you enable the integration.

This can be improved by:
1. Check if there is an existing TSS (if yes, use it).
2. Check if the API keys were changed, if yes (create a new TSS).

Reply:
I didn't want to remove the parameter. What I wanted is to never generate a random guid in the library and make the parameter required instead 😄